pkg/sql/pgwire/server.go

```diff
@@ -854,6 +884,21 @@ func (s *Server) ServeConn(
 	return nil
 }
 
+func (s *Server) lookup(ctx context.Context, ip net.IP) destinationMetrics {
+	destination := s.execCfg.IpLookup(ip)
```

Comment:
I think we discussed following the linux IP tables pattern of applying the most specific cidr-tag rule. I'd like to add a `0.0.0.0/32,public` mapping to catch all non-local, non-cloud addresses. Does this implementation support that?

Reply:
Yes - if we add that rule at the end, it will automatically match as you describe.